features/notice_permission.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Notice Permission
+
+    Scenario Outline: Check notice file read permission
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        When I run `rm -rf /run/ubuntu-advantage` with sudo
+        When I run `mkdir /run/ubuntu-advantage` with sudo
+        When I run `mkdir /run/ubuntu-advantage/notices` with sudo
+        When I run `touch /run/ubuntu-advantage/notices/crasher` with sudo
+        When I run `chmod 0 /run/ubuntu-advantage/notices/crasher` with sudo
+        When I run `pro status` as non-root
+        Then stdout matches regexp:
+        """
+        SERVICE         +AVAILABLE +DESCRIPTION
+        (anbox-cloud   +(yes|no)  +.*)?
+        ?cc-eal          +yes       +Common Criteria EAL2 Provisioning Packages
+        cis             +yes       +Security compliance and audit tools
+        esm-apps        +yes       +Expanded Security Maintenance for Applications
+        esm-infra       +yes       +Expanded Security Maintenance for Infrastructure
+        fips            +yes       +NIST-certified FIPS crypto packages
+        fips-updates    +yes       +FIPS compliant crypto packages with stable security updates
+        livepatch       +yes      +(Canonical Livepatch service|Current kernel is not supported)
+        ros             +yes       +Security Updates for the Robot Operating System
+        ros-updates     +yes       +All Updates for the Robot Operating System
+
+        For a list of all Ubuntu Pro services, run 'pro status --all'
+
+        This machine is not attached to an Ubuntu Pro subscription.
+        See https://ubuntu.com/pro
+        """
```

Comment:
This output will change over time and break this test even though the functionality we're testing here will still be working properly.

I suggest we don't check all of this output. Instead, also add a valid notice file in /run/ubuntu-advantage/notices. Then only check something like:
```
Then stdout contains substring:
"""
NOTICES
The notice that you added that we actually expect

"""
```

Reply:
Sure let me take a look at these changes!